python-sdk/src/astro/airflow/datasets.py

```diff
@@ -0,0 +1,9 @@
+try:
+    # Airflow >= 2.4
+    from airflow.datasets import Dataset
+
+    DATASET_SUPPORT = True
```

Comment:
It is probably worth to run a test in Nox, so we cover this branch

Reply:
What sort of test are you thinking about for it?